tests/security/cc/cc_audit_test_setup.pm

```diff
@@ -45,10 +45,16 @@ sub run {
     assert_script_run('sed -i \'/\[Unit\]/aStartLimitIntervalSec=0\' /usr/lib/systemd/system/auditd.service');
     assert_script_run('systemctl daemon-reload');
 
-    # modify audit rules
+    # Modify audit rules
     assert_script_run('sed -i \'s/-a task,never/#&/\' /etc/audit/rules.d/audit.rules');
     assert_script_run('systemctl restart auditd.service');
 
+    # Install certification-sles-eal4: needed by test case `crypto`
+    zypper_call('in certification-sles-eal4');
+
+    # Export MODE
+    assert_script_run("export MODE=$audit_test::mode");
```

Comment:
we already do `export MODE=` in `sub run_testcase`. And as we discussed, if we do the `export MODE=64` here, the operation `make` will miss something.

Reply:
Done.